client/state/reader/tags/selectors.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * External dependencies
+ */
+import filter from 'lodash/filter';
```

Comment:
You can now do

`import { filter, includes, sortBy, find } from 'lodash'`

instead to neaten this up :)